docs/source/contributing.md

```diff
@@ -120,6 +120,38 @@ git rebase -i HEAD~5
 
 Once your commit history is clean, ensure you have based on the [latest upstream](#updating-your-fork) before you open the PR.
 
+### Commit messages
+
+Please make the first line of your commit message a summary of the change that a user (not a developer) of Operator would like to read, 
+and prefix it with the most relevant directory of the change followed by a colon. 
```

Comment:
s/directory/area/?

Not all areas map exactly to a directory. For example, it is probably better to say "build" than "." :-)